tests/Integration/Behaviour/Features/Scenario/Product/update_options.feature

```diff
@@ -0,0 +1,123 @@
+      | name       | en-US:'The truth is out there' wallpaper |
+      | is_virtual | true                                     |
+    When I update product "product2" options with following values:
+      | visibility | show it to me plz  |
+    Then I should get error that product visibility is invalid
+    When I update product "product2" options with following values:
+      | condition | very good condition |
+    Then I should get error that product condition is invalid
+    When I update product "product2" options with following values:
+      | isbn  | isbn1                   |
+    Then I should get error that product isbn is invalid
+    When I update product "product2" options with following values:
+      | upc   | upc1                    |
+    Then I should get error that product upc is invalid
+    When I update product "product2" options with following values:
+      | ean13 | ean1                    |
+    Then I should get error that product ean13 is invalid
+    When I update product "product2" options with following values:
+      | mpn   | this is more than forty characters long string |
+    Then I should get error that product mpn is invalid
+    When I update product "product2" options with following values:
+      | reference   | invalid chars like ^;{ |
+    Then I should get error that product reference is invalid
+
+    Scenario: I update product tags in multiple languages
+      Given language "language1" with locale "en-US" exists
+      And language with iso code "en" is the default one
+      And language "language2" with locale "fr-FR" exists
+      And I add product "product3" with following information:
+        | name       | en-US:Mechanical watch;fr-FR:montre mécanique |
+        | is_virtual | false                                         |
+      And product "product3" should have following values:
+        | name       | en-US:Mechanical watch;fr-FR:montre mécanique |
+      And product "product3" localized "tags" should be "en-US:;fr-FR:"
+      When I update product "product3" options with following values:
+        | tags       | en-US:mechanic,watch;fr-FR:montre,mécanique |
+      And product "product3" localized "tags" should be "en-US:mechanic,watch;fr-FR:montre,mécanique"
+      When I update product "product3" options with following values:
+        | tags       | en-US:mechanic,watch;fr-FR: |
+      And product "product3" localized "tags" should be "en-US:mechanic,watch;fr-FR:"
+      When I update product "product3" options with following values:
+        | tags       | fr-FR:montre |
+      And product "product3" localized "tags" should be "en-US:mechanic,watch;fr-FR:montre"
+
+    Scenario: Update product tags with invalid values
+      Given product "product3" localized "tags" should be "en-US:mechanic,watch;fr-FR:montre"
+      When I update product "product3" options with following values:
+        | tags       | en-US:#<{ |
+      Then I should get error that product tag is invalid
```

Comment:
Can you add a test here to validate that the previous values are still present (not removed before the validation)